src/lib.rs

```diff
@@ -36,7 +36,6 @@ use lapin::{ConnectionProperties, ConnectionState};
 ///     }
 /// }
 /// ```
-#[derive(Debug)]
```

Comment:
To avoid breaking changes on client code, we could manually implement the `Debug` trait and just output the `amqp_address` field.

Reply:
Ok, sure, makes sense!